back-end/routes/stitchRoutes.js

```diff
@@ -130,6 +141,26 @@ router.patch('/image', async (req, res) => {
     }
 });
 
+router.patch('/exportToSpotify', async (req, res) => {
+    const { stitchId } = req.body;
+
+    if (!stitchId) {
+        return res.status(400).json({ error: 'stitchId and title are required' });
+    }
+
+    try {
+        const stitch = await prisma.stitch.findUnique({
+            where: {
+                id: parseInt(stitchId)
+            },
+        });
+
+    } catch (error) {
+        console.error('Error updating stitch:', error);
```

Comment:
Please do not commit `console.error`s. Again, this will generate logs onto the console, and act very similar to `console.log`s, so they should rather just be used for debugging.